core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java

```diff
@@ -988,18 +1031,7 @@ protected boolean isMultipartRequest(HttpServletRequest request) {
      * @return a multi part request object
      */
     protected MultiPartRequest getMultiPartRequest() {
-        MultiPartRequest mpr = null;
```

Comment:
Remnants from before the bean aliasing functionality existed I presume?

Reply:
Hm... good question, no idea :)